projects/longest-word-in-string/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Find length of the longest word in Every Language
+layout: default
+date: 2020-10-25
+last-modified: 2020-10-25
+featured-image:
+tags: [longest-word-in-string, javascript]
+authors:
+    - barhouum7
+---
+
+In this article, we'll outline the longest word in a string project.
+
```

Comment:
There should be a description section here that describes the problem in more detail with an example. I know we forgot it in your other documentation. I just wrote the following documentation as an example: https://sample-programs.therenegadecoder.com/projects/maximum-array-rotation/

Reply:
Good suggestion! I'll work on this.